.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - checkout
       - restore_cache:
           <<: *cache
-      - run: ./gradlew --no-daemon --stacktrace build -x test checkstyleTest
+      - run: ./gradlew --no-daemon --stacktrace build -x test
```

Comment:
This should be part of a different PR, in case it causes any kind of unexpected behavior.